docs/sources/choose-a-view/index.md

```diff
@@ -0,0 +1,140 @@
+---
+description: Choose a view that best suits what you need to investigate.
+keywords:
+  - Explore Profiles
+  - Concepts
+title: Choose a view
+menuTitle: Choose a view
+weight: 500
+---
+
+# Choose a view
+
+The available views in Explore Profiles correspond to how you move through your data: from wide to more specific.
```

Comment:
I'm not sure "wide" is the best word here. Maybe something like "from broader to more specific" or "from more general to more specific".

Reply:
Thank you. I've not been happy with wide but wasn't coming up with a good alternative. 
```suggestion
The available views in Explore Profiles correspond to how you move through your data: from broader to more specific.
```